### PR TITLE
Task 3 completed

### PR DESCRIPTION
**Pull request by Emidiant**
Best hyperparams: learning_rate=0.1, anneal_epochs=20, reg=1e-07
Final test accuracy: 0.89

### CODE REVIEW

**Comment by Emidiant** [comment]
Точность на лучшей модели: F1: 0.988, P: 0.980, R: 0.984, при параметрах: epochs=10, lr=1e-4, momentum=0.9

**Comment by Emidiant** [comment]
@toshiks
